src/network/networkd-radv.c

```diff
@@ -1625,3 +1634,26 @@ int config_parse_router_home_agent_lifetime(
         *home_agent_lifetime_usec = usec;
         return 0;
 }
+
+int config_parse_router_mtu(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+        Network *network = userdata;
+
+        assert(rvalue);
+
+        if (streq(rvalue, "auto"))
+                network->router_mtu_auto = true;
+        else
+                return config_parse_mtu(unit, filename, line, section, section_line, lvalue, ltype, rvalue, data, userdata);
```

Comment:
There is no way to disable emitting MTU once it is enabled. Also, the auto flag needs to be disabled when a number is specified:
```c
if (isempty(rvalue)) {
        network->router_mtu_auto = false;
        return 0;
}

if (streq(rvalue, "auto")) {
        network->router_mtu_auto = true;
        return 0;
}

r = config_parse_mtu(...);
if (r <= 0)
        return r;

network->router_mtu_auto = false;
return 0;
```